src/domain/map/components/HankeDrawer/HankeDrawer.tsx

```diff
@@ -128,9 +133,11 @@ const HankeDrawer: React.FC<React.PropsWithChildren<Props>> = ({
         // If original hankealue feature has different work area features than modified feature,
         // revert back to original geometry
         modifiedFeature.setGeometry(originalFeature.getGeometry());
+      } else {
+        setSelectedFeature(modifiedFeature as Feature<Geometry>);
```

Comment:
SonarCloud says that "This assertion is unnecessary since it does not change the type of the expression."

https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=1171&id=City-of-Helsinki_haitaton-ui&open=AZY-sd0Q9sQiuKmwq-AS